From: "Igor B. Poretsky" <poretsky@mlbox.ru>
Date: Tue, 30 Jun 2026 05:06:20 +0300
Subject: Don't trip over cruft left by vm

Somewhere before vm 8.2.0b-5, that package didn't clean up properly after
itself, causing emacspeak to fail during byte compiling. Avoid the failure.
---
 lisp/emacspeak-vm.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/emacspeak-vm.el b/lisp/emacspeak-vm.el
index 7b27931..30592da 100644
--- a/lisp/emacspeak-vm.el
+++ b/lisp/emacspeak-vm.el
@@ -49,7 +49,7 @@
 ;;{{{ requires
 (cl-declaim  (optimize  (safety 0) (speed 3)))
 (require 'emacspeak-preamble)
-(require  'vm "vm" 'no-error)
+(with-demoted-errors "Error loading VM: %s" (require 'vm () 'no-error))
 
 ;;}}}
 ;;{{{ Forward Decls:
